index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Survey Form</title>
+</head>
+
+<body>
+  <header>
+    <h1>Survey Form</h1>
+    <p>Please enjoy this sick survey form</p>
+    <h6>(I worked really hard on it)</h6>
+  </header>
+  <main>
+    <form action="" method="GET">
+      <section>
+        <label for="name">Name</label>
+        <input type="text" name="name" required><br>
+      </section>
+      <section>
+        <label for="email">Email</label>
+        <input type="text" name="email" required><br>
```

Comment:
The `<br>` tag that you're using here to separate things is likely just presentational since there's no semantic necessity to having the labels and inputs on different lines (i.e. if the label is on the same line as the input it still means the same thing).

If you need your inputs and labels on different lines for presentational reasons, CSS is the right tool for that.